### PR TITLE
Rewrites some Orion Trail grammar

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
spellcheck: Various grammar in the Orion Trail arcade game has been tweaked.
/:cl:

Whoever added spaceport raids and changeling infiltration was evidently very tired, as a lot of the grammar was improper or messy. I fixed up all of the improper grammar in those cases, including capitalization and punctuation, in order to cement my title as a professional bikeshedder, as well as replacing some messages with fancy new ones such as:

"The changeling(s) run up to (target name) and capitulates them!" 
(sounds really weird, and capitulates means "surrenders" )
has been turned into
"(a changeling's name) decapitates (target name) in a single cleaving arc!"
(spooky!)

### CODE REVIEW

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -1044,8 +1051,8 @@
 	emagged = TRUE
 
 /mob/living/simple_animal/hostile/syndicate/ranged/orion
-	name = "Spaceport Security"
-	desc = "The Premier security forces for all spaceports found along the Orion Trail."
+	name = "spaceport security"
+	desc = "Premiere corporate security forces for all spaceports found along the Orion Trail."
```
Comment (RemieRichards) [inline code/game/machinery/computer/arcade.dm]: No, the word is Premier.

File: code/game/machinery/computer/arcade.dm [hunk 2]
```diff
@@ -870,18 +871,24 @@
 					var/chancetokill = 30*lings_aboard-(5*alive) //eg: 30*2-(10) = 50%, 2 lings, 2 crew is 50% chance
 					if(prob(chancetokill))
 						var/deadguy = remove_crewmember()
-						eventdat += "<br>The Changeling[ling2 ? "s":""] run[ling2 ? "":"s"] up to [deadguy] and capitulates them!"
+						var/murder_text = pick("The changeling*LINGS?* bring down [deadguy] and disembowel*PLURAL?* them in a spray of gore!", \
+						"[ling2 ? pick(ling1, ling2) : ling1] corners [deadguy] and impales them through the stomach!", \
+						"[ling2 ? pick(ling1, ling2) : ling1] decapitates [deadguy] in a single cleaving arc!")
+						murder_text = replacetext(murder_text, "changeling*LINGS?*", ling2 ? "changelings" : "changeling")
```
Comment (RemieRichards) [inline code/game/machinery/computer/arcade.dm]: the original method is better.

**Comment by RemieRichards** [comment]
>Whoever added spaceport raids and changeling infiltration was evidently very tired

Hmph.

File: code/game/machinery/computer/arcade.dm [hunk 3]
```diff
@@ -895,17 +902,17 @@
...
+				eventdat += "The spaceport is on high alert! You've been barred from docking by the local authorities after your failed raid."
 				if(last_spaceport_action)
-					eventdat += "<br>Last Spaceport Action: [last_spaceport_action]"
+					eventdat += "<br><b>Last Spaceport Action:</b> [last_spaceport_action]"
 				eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];leave_spaceport=1'>Depart Spaceport</a></P>"
 				eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];close=1'>Close</a></P>"
 			else
-				eventdat += "You pull the ship up to dock at a nearby Spaceport, lucky find!"
-				eventdat += "<br>This Spaceport is home to travellers who failed to reach Orion, but managed to find a different home..."
+				eventdat += "Your jump into the sector yields a spaceport - a rare and lucky find!"
```
Comment (RemieRichards) [inline code/game/machinery/computer/arcade.dm]: but it's not rare, it's an average event

**Comment by CosmicScientist** [comment]
Just a teensy (very big) request. Could you fix when it goes to game over without explanation?

I assume it's when you run out of fuel or perhaps crew through an event but it skips to "You drift forever, unable to reach Orion."

File: code/game/machinery/computer/arcade.dm [hunk 4]
```diff
@@ -609,17 +608,15 @@
...
 					if(isliving(usr))
 						var/mob/living/L = usr
 						L.Stun(200, ignore_canstun = TRUE) //you can't run :^)
 					var/S = new /obj/singularity/academy(usr.loc)
-					emagged = FALSE //immediately removes emagged status so people can't kill themselves by sprinting up and interacting
-					sleep(50)
-					say("[S] winks out, just as suddenly as it appeared.")
-					qdel(S)
+					QDEL_IN(src, 50)
+					addtimer(CALLBACK(src, /atom/movable/proc/say, "[S] winks out, just as suddenly as it appeared."), 50)
```
Comment (Cyberboss) [inline code/game/machinery/computer/arcade.dm]: Reverse these two lines so it runs the say before it's deleted

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Pull request merged**